source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js

```diff
@@ -195,12 +210,12 @@
         this.period = data.Period;
         this.currentPosition= data.CurrentPosition || '';
         this.seniority = data.Seniority || '';
-        this.evaluators = data.Evaluators;
+        this.evaluators(data.Evaluators);
         this.evaluatorsString = ko.computed(function () {
-            return this.evaluators.toString().replace(/,/g, ', ');
+            return this.evaluators().toString().replace(/,/g, ', ');
```

Comment:
Note that Array.prototype.toString() is just a call to Array.prototype.join(',');

You can achieve the same thing and avoid the replace by using join:

```javascript
return this.evaluators().join(', ');
```

Sources:
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toString
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join